scripts/files/file_tiff.py

```diff
@@ -1,11 +1,14 @@
 import json
+from decimal import Decimal
 from enum import Enum
 from typing import Any, Dict, List, Optional
 from urllib.parse import unquote
 
 from scripts.gdal.gdal_helper import GDALExecutionException, gdal_info, run_gdal
 from scripts.gdal.gdalinfo import GdalInfo
 
+DEFAULT_NO_DATA_VALUE = Decimal(-9999)
```

Comment:
Not part of this PR but if you want you could add a comment about why we are using `-9999`

Reply:
Can do! Do you have a reference? I think it was mentioned recently, but I can't find it in Slack.

Reply:
#901 is in draft, because it depends on this PR.